chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/endpoints/dto/AssaultPropertiesUpdate.java

```diff
@@ -83,7 +87,7 @@
   @DecimalMin("0.1")
   private Double cpuLoadTargetFraction;
 
-  @Nullable private String runtimeAssaultCronExpression;
+  @Nullable private String cpuCronExpression;
```

Comment:
When we avoid having null values (see previous comment), we can remove the `@Nullable` annotation

Reply:
`@Nullable` on `AssaultPropertiesUpdate` should always stay, as a null value here means "don't change"

Reply:
true